components/server-side-render/README.md

```diff
@@ -0,0 +1,24 @@
+ServerSideRender
+=======
+
+ServerSideRender component is used for server-side-rendering preview in Gutenberg editor, specifically for dynamic blocks.
+
+
+## Usage
+
+Render core/latest-posts preview.
+```jsx
+	<ServerSideRender
+		block="core/latest-posts"
```

Comment:
We should probably register another block for something that makes more sense server-rendered (maybe tag cloud widget?) to use as the example.

Reply:
I could add `Archives` block from https://github.com/WordPress/gutenberg/pull/5495. It's otherwise ready except for the preview which was waiting for SSR.

Reply:
@mtias For usage example I merged the code from this PR to #5495 which adds `Archives` block -- thought that this way this PR wouldn't have to wait for potential fixes for the `Archives` block.

So basically #5495 now includes adding Archives block + SSR component with the idea that SSR component could be merged separately first.

Let me know if that's OK or if you'd prefer having the Archives block added here under this PR.

cc @westonruter

Reply:
@mtias Updated the description with added screenshots of [Archives block based on SSR](https://github.com/WordPress/gutenberg/pull/5495). Let me know if there's anything else you see in the PR that requires addressing.